lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -245,6 +251,12 @@ class AzurePlatformSchema:
             validate=validate.Regexp(constants.GUID_REGEXP),
         ),
     )
+    cloud: str = field(
+        default="public",
+        metadata=field_metadata(
+            validate=validate.OneOf(["mooncake", "public", "germany", "fairfax"]),
```

Comment:
Don't create other names, use names in the SDK like "AzureChinaCloud".

Reply:
updated